src/utils/exportServerQueryBuilder.ts

```diff
@@ -15,8 +15,8 @@ export async function queryBulkDataServer(url: string): Promise<{ output: BulkDa
   try {
```

Comment:
I think we should update the type for the output of `queryBulkDataServer()` to be `Promise<{ output: BulkDataResponse[]}>` since the error is no longer included in the object and is simply thrown instead.